### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by 0xIsho**
### PR Prelude

Thank you for working on SpaceVim! :)

Please complete these steps and check these boxes before filing your PR:

- [x] I have read and understood SpaceVim's [CONTRIBUTING](https://github.com/SpaceVim/SpaceVim/blob/master/CONTRIBUTING.md) document.
- [x] I have read and understood SpaceVim's [CODE_OF_CONDUCT](https://github.com/SpaceVim/SpaceVim/blob/master/CODE_OF_CONDUCT.md) document.
- [x] I understand my PR may be closed if it becomes obvious I didn't actually perform all of these steps.

### Why this change is necessary and useful?

I've fixed plenty of grammar mistakes, rephrased some stuff, and tried to improve the docs in general.

Please review the changes and let me know if anything needs adjustment.

Still working on the rest of the docs.

### CODE REVIEW

File: bundle/vim-matchup/README.md [hunk 1]
```diff
@@ -713,7 +713,7 @@ surround replacement rules but is not currently supported.
 
 ### Auto-closing plugins
 
-match-up does not provide auto-complete or auto-insertion of matches.
+match-up does not provides auto-complete or auto-insertion of matches.
```
Comment (wsdjeg) [inline bundle/vim-matchup/README.md]: It should be `does not provide`.

Comment (0xIsho) [reply bundle/vim-matchup/README.md]: I should pay more attention to where i'm `sed`ing from now on :/

**Comment by 0xIsho** [comment]
I've done the fixes for most of the docs. `docs/posts/` and `docs/api/` Still needs reviewing (Didn't work on it yet!). 
I've also cleaned up the commit history (squashed some commits).

I think the branch can be merged (after review, of course) as-is. or after i review the remaining pages (?).

File: bundle/spacevim/autoload/spacevimcn.vim [hunk 2]
```diff
@@ -45,10 +45,6 @@ scriptencoding utf-8
 " <
 let g:spacevim_default_indent          = 2
 
-""
-" 在输入模式下，使用空格替代 `<Tab>`
-let g:spacevim_expand_tab              = 1
-
```
Comment (wsdjeg) [inline bundle/spacevim/autoload/spacevimcn.vim]: This should not be removed.

File: docs/community.md [hunk 3]
```diff
@@ -11,6 +11,10 @@ Visit these resources for help or general questions.
 <!-- vim-markdown-toc GFM -->
 
 - [News](#news)
+<<<<<<< HEAD
+=======
+- [Bug reports and feature requests](#bug-reports-and-feature-requests)
+>>>>>>> c55c445e (Fix grammar)
```
Comment (wsdjeg) [inline docs/community.md]: conflict should be fixed.

File: wiki/cn/Following-HEAD.md [hunk 4]
```diff
@@ -7,6 +7,69 @@
 <!-- call SpaceVim#dev#followHEAD#update('cn') -->
 <!-- SpaceVim follow HEAD en start -->
 
+#### 新特性
```
Comment (wsdjeg) [inline wiki/cn/Following-HEAD.md]: this should be cleared.

File: wiki/en/Following-HEAD.md [hunk 5]
```diff
@@ -9,6 +9,68 @@ The next release is v1.8.0:
 <!-- call SpaceVim#dev#followHEAD#update('en') -->
 <!-- SpaceVim follow HEAD en start -->
 
+#### New Features
```
Comment (wsdjeg) [inline wiki/en/Following-HEAD.md]: this should be removed.

**Review by wsdjeg — APPROVED** [review]

**Pull request merged**